src/index.js

```diff
@@ -15,5 +16,6 @@ export {
   CidInfo,
   IpldGraph,
   ObjectInfo,
-  exploreBundle
+  exploreBundle,
+  heliaBundle
```

Comment:
exporting helia bundle now so we don't have to duplicate code in explore.ipld.io..

though i will need to figure out a decent solution for webui & desktop, since IPFS code in this package is now dependent upon `helia.blockstore.*` calls. 

We should probably accept a few configurable callback functions that consumers can pass in to get blocks, get identity, and other things we need to do. If those aren't passed, we should stick with current functionality from this PR.